sdk/src/manifest_store.rs

```diff
@@ -302,4 +333,19 @@ mod tests {
         assert_eq!(manifest.issuer().unwrap(), "C2PA Test Signing Cert");
         assert!(manifest.time().is_some());
     }
+
+    #[cfg_attr(not(target_arch = "wasm32"), actix::test)]
+    #[cfg_attr(target_arch = "wasm32", wasm_bindgen_test)]
+    async fn manifest_report_from_manifest_and_asset_bytes_async() {
+        let asset_bytes = include_bytes!("../tests/fixtures/cloud.jpg");
+        let manifest_bytes = include_bytes!("../tests/fixtures/cloud_manifest.c2pa");
+
+        let manifest_store =
+            ManifestStore::from_manifest_and_asset_bytes_async(manifest_bytes, asset_bytes)
+                .await
+                .unwrap();
+        assert!(!manifest_store.manifests().is_empty());
+        assert!(manifest_store.validation_status().is_none());
+        println!("{}", manifest_store);
```

Comment:
Do we want to print the manifest store as part of this test? Not sure if it will cloud test output.